documentation/service-rules/ecf1/lead-provider-api.md

```diff
@@ -2,4 +2,247 @@
+        "declaration_type": "started",
+        "declaration_date": "2020-11-13T11:21:55Z",
+        "course_identifier": "ecf-induction",
+        "state": "eligible",
+        "updated_at": "2020-11-13T11:21:55Z",
+        "created_at": "2020-11-13T11:21:55Z",
+        "delivery_partner_id": "99ca2223-8c1f-4ac8-985d-a0672e97694e",
+        "statement_id": "99ca2223-8c1f-4ac8-985d-a0672e97694e",
+        "clawback_statement_id": null,
+        "ineligible_for_funding_reason": null,
+        "mentor_id": "907f61ed-5770-4d38-b22c-1a4265939378",
+        "uplift_paid": true,
+        "evidence_held": "other",
+        "has_passed": null,
+        "lead_provider_name": "Example Institute"
+      }
+    }
+  ]
+}
+```
+
+Also lead providers have real-time status updates of a specific declaration via the endpoint GET `api/v3/participant-declarations/{id}`.
+
+### How do lead providers check which statement a declaration has been linked to?
+
+Lead providers can check for updates on the statement a declaration has been linked to and its status, via the endpoint GET `/api/v3/statements/{id}`, using the `statement_id` from the declaration response above as `{id}`.
+
+### How do lead providers know if a declaration has been successfully submitted and processed?
+
+A valid declaration will return a 200 response whereas an invalid declaration request will return an error message.
+
+Many validations are in place to guarantee a declaration is correctly created:
+
+#### 1. Automatic Checks Before Validation
+- The system verifies whether the selected course is still supported.
+- It tracks declaration attempts to prevent unnecessary or duplicate submissions.
+
+#### 2. Required Information
+- **Declaration Date** – Must be provided; otherwise, an error message is shown.
+- **Declaration Type** – Must be provided; otherwise, an error message is shown.
+
+#### 3. Participant Requirements
+- The participant must have a valid identity in the system.
+- The participant must not have withdrawn from the program.
+
+#### 4. Date Validations
+- The **declaration date** must be a valid date.
+- The **declaration date** cannot be in the future.
+- The **declaration date** must be within each declaration type milestone start dates.
+
+#### 5. Course & Provider Validations
+- The **course identifier** must be valid.
+- The **lead provider** must be correctly linked to the participant.
+
+#### 6. Additional Compliance Checks
+- The system checks if the required **statement** is available.
+- A **valid milestone** must exist for the declaration.
+- The system checks for duplicated declarations.
+- If the participant is a **mentor**, they can only declare a `started` or `completed` status.
+- The provided **evidence** must meet the necessary requirements.
+
+These validations help ensure accurate and compliant record declarations while preventing errors in the process.
+
+#### Possible validation error messages:
```

Comment:
I think there are some extra validation messages you can get from the pulled in validators, for example `DeclarationDateValidator` for milestone dates matching up to declaration date